ansible_base/activitystream/models/entry.py

```diff
@@ -63,6 +63,13 @@ class Meta:
     )
 
     def __str__(self):
+        # Enhanced display for RBAC role assignments
+        if self.content_type and self.content_type.model.lower() in ['roleuserassignment', 'roleteamassignment']:
+            operation_text = self.get_operation_display()
+            created_by_text = str(self.created_by) if self.created_by else "Unknown"
+            return f'[{self.created}] Role assignment {operation_text.lower()} by {created_by_text}'
```

Comment:
I agree a custom string would make sense because "roleuserassignment" is verbose. But I would question whether we could solve this generally by using `_meta.verbose_name.title()` instead? Because "Role assignment" drops the user/team designation, which could be useful. This also appears to drop the object_id. I don't think content_type (as a string) was ever useful but it dropped that too.